### DISCUSSION TITLE
Show number of zombie processes

### DISCUSSION THREAD

**Discussion by myin142** [netdata/netdata#12133] [Q&A] [answered]
I'm not sure if I'm missing something or if that's something unimportant, but isn't it possible to show the number of zombie processes on the system? It's only possible to see how many are running but not in what state they are.

Currently I am using the node-exporter of prometheus to get that information. They list the number of processes in each state. But it would be great if I don't have to start another service just to get that one information.

**Comment by ilyam8** [comment, marked as answer]
Implemented in #12305

---

Hi, @myin142. The answer, unfortunately, is - Netdata doesn't collect the number of zombie processes.

I completely agree that we need to add that. That would be an update to the apps.plugin cc @vlvkobal

**Reply by ilyam8** [reply]
FR is #12139

**Comment by ilyam8** [comment]
@myin142 can you create a [feature request](https://github.com/netdata/netdata/issues/new?assignees=&labels=feature+request%2Cneeds+triage&template=FEAT_REQUEST.yml&title=%5BFeat%5D%3A+)?

**Reply by cpipilas** [reply]
@myin142 I like the idea as well! Maybe it would be useful to find the zombie's parent as well.
